test/features/cloudnative/codemodules/codemodules.go

```diff
@@ -203,7 +206,7 @@ func WithProxyCA(t *testing.T, proxySpec *value.Source) features.Feature {
 		dynakubeComponents.WithIstioIntegration(),
 		dynakubeComponents.WithProxy(proxySpec),
 		dynakubeComponents.WithAnnotations(map[string]string{
-			dynakube.AnnotationFeatureActiveGateAutomaticTLSCertificate: "true",
```

Comment:
shouldn't this stay `true`? (or it can be removed)

OR `WithProxyAndAutomaticAGCert` supposed to "replace" it

Reply:
WithProxyCA is not needed, removed